pandas/tests/indexes/ranges/test_range.py

```diff
@@ -167,6 +167,10 @@ def test_cache(self):
         idx.any()
         assert idx._cache == {}
 
+        for _ in idx:
+            ...
```

Comment:
i think "pass" would be more idiomatic here

Reply:
Ok